PetGame/PetGame/Washing/ProgressBar.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ProgressBar.swift
+//  PetGame
+//
+//  Created by Rennick Carter M. on 4/11/19.
+//  Copyright © 2019 Rennick Carter M. All rights reserved.
+//
+
+import SpriteKit
+
+class ProgressBar : SKSpriteNode{
+    private var fillValue : CGFloat //How full the progress bar is
+    private var filledWidth : CGFloat //How wide the progress bar is when completely filled
+    
+    init(textureName: String){
+        let texture = SKTexture.init(imageNamed: textureName)
+        fillValue = 1.0
+        filledWidth = 1.0 //Fake initialization, actual value is set later
```

Comment:
if you define the variable as optional you wont need to do this.
aka 
```
private var fillValue: CGFloat?
```
you wont get the compiler error saying you haven't defined all the values before calling super etc.